EKF/covariance.cpp

```diff
@@ -734,12 +735,12 @@ void Ekf::fixCovarianceErrors(bool force_symmetry)
 
 	for (int i = 4; i <= 6; i++) {
 		// NED velocity states
-		P(i,i) = math::constrain(P(i,i), 0.0f, P_lim[1]);
+		P(i,i) = math::constrain(P(i,i), 1E-6f P_lim[1]);
```

Comment:
Missing a comma here
```suggestion
		P(i,i) = math::constrain(P(i,i), 1E-6f, P_lim[1]);
```